src/roundElements.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Rounds all elements of the given matrix using the given precision
+ * @param m  {{a: number, b: number, c: number, d: number, e: number, f: number}} a matrix to round
+ * @param precision a precision to use for Math.round. Defaults to 10000000000 (meaning which rounds to the 10th digit after the comma).
+ * @returns {{a: number, b: number, c: number, d: number, e: number, f: number}} the rounded matrix
+ */
+
+export function roundElements (m, precision) {
```

Comment:
Can we rename it with smoothMatrix? Thought?

Reply:
Sure!